app/views/users/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1> Displays bio data of a given user</h1>
\ No newline at end of file
+<ul class="users_container">
+ <%= render "shared/user_profile", user: @user %>
+  <div class="user_bio">
+    <h3>Bio</h3>
+    <p><%= @user.Bio %></p>
+  </div>
+  <% @user.three_most_recent_post.each_with_index do |posts, index | %>
+    <li>
+      <div class="user_bio">
+        <h3>Post#<%= index+1 %></h3>
+        <p><%= posts.Text %></p>
+        <div class="user_recent_posts">
+          <span>
+            Comments:<%= posts.Comments_counter.nil? ? '0' : posts.Comments_counter %>,
+          </span>
+          <span>
+            Likes:<%= posts.Likes_counter.nil? ? '0' : posts.Likes_counter %>
+          </span>
+        </div>
+
+      </div>
```

Comment:
- [ ] Great job on your implementation so far! 🚀 However, there are a few minor issues that need some attention to ensure your code functions as expected. 

   - It appears that the data from the database isn't displaying correctly in the UI. This issue arises from the way you're referencing your data column names. I've noticed that you're using capital letters at the beginning of the column names, but they should match how they are saved in the database, typically with lowercase letters. 

   For example, in the highlighted code, you can access the `bio` from your database like this:

   ```ruby
     <p><%= @user.bio %></p>
   ``` 

   Instead of:

   ```ruby
      <p><%= @user.Bio %></p>
   ``` 

   - Kindly make sure to make this correction across your entire codebase. Some of the column names you need to correct in this file includes: `bio`, `text`,  `likes_counter`, and `comments_counter`.

- [ ] Also, kindly consider correctly the spelling mistake on the method name `three_most_recent_post` as it's supposed to be `three_most_recent_posts`.

Happy coding! 😊👨‍💻🚀